cli/internal/helm/loader.go

```diff
@@ -405,6 +431,14 @@ func (i *ChartLoader) loadConstellationServicesValues() (map[string]any, error)
 		values["tags"] = map[string]any{
 			"AWS": true,
 		}
+		values["aws-load-balancer-controller"] = map[string]any{
+			// TODO: nodeSelector is already set in (customized) values.yaml. Should we have a custom values.yaml instead of configuring here for easier testing?
```

Comment:
It seems like `nodeSelector` always has the same value. Thus it should be placed in values.yaml. 
So far we have kept all static values in values.yaml.